### PR TITLE
WW - Setup

### PR DESCRIPTION
**Pull request by Alixers505**
Per Brandon, only pushing up the basic setup.

Custom website sections are proving difficult. Will look at those as a separate branch down the road.

### CODE REVIEW

File: sites/waterworld/config/site.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  gtm: {
+    containerId: 'GTM-MXBGH3V',
+  },
+  wufoo: {
+    userName: 'cygnuscorporate',
+  },
+  magazines: {
+    description: 'WaterWorld serves professionals in the North American municipal water/wastewater industry. WaterWorld delivers monthly news and information about products and services, technology, applications, legislation and regulations to help you successfully plan, design, operate and maintain your water/wastewater systems. Editorial topics include: drinking water treatment, wastewater treatment, filtration, disinfection, membrane systems, biosolids and sludge treatment, water reuse, pipe maintenance and repairs, trenchless technology, meter reading, valves, pumps, stormwater management, odor control, computers and automation technology, corrosion control, and so much more!',
+  },
+};
```
Comment (B77Mills) [inline sites/waterworld/config/site.js]: Can you also port the contactUs portion of the site config.

Comment (Alixers505) [reply sites/waterworld/config/site.js]: Ah, this wasn't set up on the example site, so I assumed it was being input in a different way. Should the information be laid out exactly as listed in the previous site config?

**Comment by Alixers505** [comment]
@B77Mills -- if that was the update you were requesting, please confirm and I will update it on the other sites I replicated as well.

File: sites/waterworld/config/site.js (hunk 1 above)
Comment (B77Mills) [reply sites/waterworld/config/site.js]: @Alixers505 it just got merged into the other ones.  
WW should be 
```
contactUs: {
    branding: {
      bgColor: '#0072bc',
      logo: 'https://base.imgix.net/files/base/pennwell/ww/logo.png?h=60',
    },
    to: 'abailie@endeavorb2b.com',
  },
```

**Comment by Alixers505** [comment]
Going to close down this PR and start a new branch over, as it seems to have revolving issues.

**Pull request closed without merging**